umh.docs.umh.app/content/en/docs/features/connectivity/additionalconnectivity/ifm-retrofitting.md

```diff
@@ -40,7 +40,7 @@ To use ifm IO-link gateways and [Sensorconnect](/docs/architecture/microservices
 
 1. Ensure all IO-Link gateways are in the same network or accessible from your instance of the United Manufacturing Hub.
 2. Retrofit the machines by connecting the desired sensors and establish a connection with ifm IO-Link gateways.
-3. Configure the [Sensorconnect IP-range](/docs/architecture/helm-chart/#sensor-connect) to either match the IP address using subnet notation /32, or, in cases involving multiple masters, configure it to scan an entire range, for example /24. To change the value, go to the [Customize the United Manufacturing Hub](/docs/production-guide/administration/customize-umh-installation/) section.
+3. Configure the [Sensorconnect IP-range](/docs/architecture/helm-chart/#sensor-connect) to either match the IP address using subnet notation /32, or, in cases involving multiple masters, configure it to scan an entire range, for example /24. To change the value, go to the [Helm Chart](/docs/reference/helm-chart/) section.
 4. Once completed, the data should be available in your [Unified Namespace](/docs/features/unified-namespace/).
 
```

Comment:
Also we should explain with kubectl commands how to configure the sensorconnect IP range

When helm upgrading, we need to make sure it uses the same version and does not also upgrade the umh version. maybe instead of refering to customize-umh-installation, we could refer to the get started area and add it there under managing?

Reply:
Use `$(which helm)` instead of just `helm`

Reply:
updated